packages/ddp-client/livedata_connection_tests.js

```diff
@@ -2,7 +2,10 @@ var newConnection = function (stream) {
   // Some of these tests leave outstanding methods with no result yet
   // returned. This should not block us from re-running tests when sources
   // change.
-  return new LivedataTest.Connection(stream, {reloadWithOutstanding: true});
+  return new LivedataTest.Connection(stream, {
+    reloadWithOutstanding: true,
+    bufferedWritesInterval: 0
```

Comment:
I wrote such a test: https://github.com/meteor/meteor/commit/74230beba8777cb9104c4180525e2a3fe09772c4

Appreciate people's thoughts: 
 - Is this testing the right thing?
 - Is it sufficiently thorough or do we want to test more stuff (other combination of message timings, other data messages, not just in the stub?)
 - Is there any way to deterministically know when the max age is hit? It was hitting it at inconsistent/unexpected times for me, which I put down to the content of the test taking relatively significant time to run, as you can see in my comments.